src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/RawlsDAO.scala

```diff
@@ -19,7 +21,7 @@ import scala.concurrent.Future
   */
 
 object RawlsDAO {
-  lazy val serviceName = "Rawls"
+  lazy val serviceName = health.Subsystems.Rawls
```

Comment:
looks like if you import import org.broadinstitute.dsde.workbench.util.health.Subsystems then health wont be needed and will follow the other pattern? (but its possible I am missing something too)

Reply:
I don't know why intellij wanted to do it this way